main.cpp

```diff
@@ -410,9 +424,21 @@ int main()
 	Example::main();
     MidiKeyboard keyboard;
     keyboard.printKeys();
+    //Midikeyboard project 5 part2
+    std::cout << keyboard.pitch << " is the pitch" << std::endl;
+    keyboard.mkFunc();
+    //
```

Comment:
your member function prints out the address of `keyboard`.   
Can you make your main() function also print out the address? 
- [ ] revise